src/main/webapp/WEB-INF/tags/instructor/results/participantGroupBySecondaryParticipantPanel.tag

```diff
@@ -9,46 +9,46 @@
+          <strong>${fn:escapeXml(groupByParticipantPanel.name)}</strong>
+          <img src="" alt="No Image Given" class="hidden profile-pic-icon-hidden">
+          <a <c:if test="${not empty groupByParticipantPanel.secondaryParticipantPanels}">class="link-in-dark-bg"</c:if> href="mailto:${groupByParticipantPanel.participantIdentifier}">[${groupByParticipantPanel.participantIdentifier}]</a>
         </div>
-    </div>
-    <div class="panel-collapse collapse in">
-        <div class="panel-body">
-            <c:choose>
-                <c:when test="${not empty groupByParticipantPanel.secondaryParticipantPanels}">
-                    <c:forEach items="${groupByParticipantPanel.secondaryParticipantPanels}" var="secondaryParticipantPanel" varStatus="i">
-                        <results:secondaryParticipantPanel secondaryParticipantPanelBody="${secondaryParticipantPanel}"
-                                                           primaryParticipantPanel="${groupByParticipantPanel}"
-                                                           secondaryParticipantIndex="${i.index}"/>
-                     </c:forEach>
-                 </c:when>
-                 <c:otherwise>
-                    <i>There are no responses ${groupByParticipantPanel.giver? 'given' : 'received'} by this user or you may not have the permission to see the response</i>
-                 </c:otherwise>
-             </c:choose>
+      </c:when>
+      <c:otherwise>
+        <div class="inline panel-heading-text">
+          <strong>${fn:escapeXml(groupByParticipantPanel.name)}</strong>
         </div>
+      </c:otherwise>
+    </c:choose>
+    <div class="pull-right">
+      <c:if test="${not empty groupByParticipantPanel.moderationButton}">
+        <results:moderationButton moderationButton="${groupByParticipantPanel.moderationButton}" />
+      </c:if>
+      &nbsp;
+      <div class="display-icon" style="display:inline;">
+        <span class='glyphicon glyphicon-chevron-up pull-right'></span>
+      </div>
+    </div>
+  </div>
+  <div class="panel-collapse collapse in">
+    <div class="panel-body">
+      <c:choose>
+        <c:when test="${not empty groupByParticipantPanel.secondaryParticipantPanels}">
+          <c:forEach items="${groupByParticipantPanel.secondaryParticipantPanels}" var="secondaryParticipantPanel" varStatus="i">
+            <results:secondaryParticipantPanel secondaryParticipantPanelBody="${secondaryParticipantPanel}"
+                primaryParticipantPanel="${groupByParticipantPanel}"
+                secondaryParticipantIndex="${i.index}"/>
+           </c:forEach>
+         </c:when>
+         <c:otherwise>
+          <i>There are no responses ${groupByParticipantPanel.giver? 'given' : 'received'} by this user or you may not have the permission to see the response</i>
+         </c:otherwise>
+       </c:choose>
```

Comment:
Remove a space from `</c:forEach>` to `</c:choose>`, except for the line with `<i>`.